packages/db-model/src/db.ts

```diff
@@ -291,215 +187,14 @@ export async function query<T>(
  * @param {Knex.Transaction} [trx]
  * @returns {Promise<T[]>}
  */
-export async function batchInsert<T>(
+export function batchInsert<T>(
   connection: Knex,
   table: string,
   data: object[],
   chunkSize: number,
   trx?: Knex.Transaction
-): Promise<T[]> {
-  const qb = queryBuilder(connection, trx);
-  const result = await qb.batchInsert(table, object.toSnakeCase(data), chunkSize).returning('*');
-
-  return object.toCamelCase(result);
-}
-
-/**
- * Execute SQL raw query returning a scalar value.
- *
- * Example:
- *  const count:number = await db.getValue<number>(
- *    connection,
- *    'SELECT count(id) FROM jobs WHERE is_active = 1'
- *  );
- *
- * @param {Knex} connection
- * @param {string} sql
- * @param {any} params
- * @param {Knex.Transaction} trx
- * @returns {Promise<T | null>}
- */
-export async function getValue<T>(
-  connection: Knex,
-  sql: string,
-  params?: any,
-  trx?: Knex.Transaction
-): Promise<T | null> {
+): Knex.QueryBuilder {
   const qb = queryBuilder(connection, trx);
-  const [data] = await qb.raw(sql, params);
-
-  if (!data) {
-    return null;
-  }
-
-  const [value] = Object.values<T>(data);
-
-  return value;
-}
-
-/**
- * Execute SQL raw query returning a boolean result.
- *
- * Example:
- *  const result:boolean = await db.check(
- *    connection,
- *    'SELECT is_active FROM jobs WHERE id = :jobId'
- *  );
- *
- * @param {Knex} connection
- * @param {string} sql
- * @param {any} params
- * @param {Knex.Transaction} trx
- * @returns {Promise<boolean>}
- */
-export async function check(connection: Knex, sql: string, params?: any, trx?: Knex.Transaction): Promise<boolean> {
-  const value = await getValue<boolean>(connection, sql, params, trx);
```

Comment:
These are pretty common and handy utilities that serve good use cases in small to larger projects - any specific reason to remove them?

* `getValue<T>` - You need a single scalar value from a query w/o caring what the column name is.
* `getJson<T>` - You're fetching a non-tabular resultset, serialized into JSON string by the DBMS and the App needs the deserialised object -- PG / MSSQL
* `check` - You're checking something and expect the boolean output out from the query. 
* `invoke` / `exec` - When you're app needs to make use of the procedures, functions defined in the query in a more intuitive way. 
* etc.